test/docs.js

```diff
@@ -78,6 +78,7 @@ describe('documentation', function() {
         gcloud: gcloud,
         require: require,
         process: process,
+        console: console,
```

Comment:
We'll have to overwrite these methods if we don't want the docs test to explode in our terminal when run. :)